src/EditorFeatures/Test2/Classification/ClassificationTests.vb

```diff
@@ -155,8 +157,9 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Classification
                 Dim document = workspace.CurrentSolution.Projects.Single().Documents.Single()
                 Dim text = Await document.GetTextAsync()
 
+                Dim spanArray As TextSpan() = {New TextSpan(0, text.Length)}
                 Dim spans = Await ClassifierHelper.GetClassifiedSpansAsync(
-                    document, New TextSpan(0, text.Length), ClassificationOptions.Default, includeAdditiveSpans:=False, CancellationToken.None)
+                    document, spanArray, ClassificationOptions.Default, includeAdditiveSpans:=False, CancellationToken.None)
```

Comment:
same feedback.

Reply:
and all the cases below.

also, consider having an extension that does this.  that way you wouldn't have to update these use sites.